### ISSUE TITLE
No circle is drawn on a line chart when there is only one data. (version 3.2.2 only)

### ISSUE THREAD

**Issue by mock123** [danielgindi/Charts#3848]
## What did you do?

I updated the Charts library from 3.2.1 to 3.2.2.

## What did you expect to happen?

When a line chart has just "one" entry data, the chart shows a circle to indicate the data.

#### v3.2.1 (with highlight /  It draws a circle. correct.)
<img src="https://user-images.githubusercontent.com/36657040/52917368-f183c380-332d-11e9-813f-2511d7dceeb9.jpg" width="150">

#### v3.2.1 (without highlight / It draws a circle. correct.)
<img src="https://user-images.githubusercontent.com/36657040/52917369-f183c380-332d-11e9-88b2-3cfc8013bade.jpg" width="150">

## What happened instead?
No circle was drawn on the chart.
*`chartDataSet.drawCirclesEnabled` is always set as TRUE.

The problem happens when there is only "one" entry. 
(But if there are more than two entries, it works fine.)

#### v3.2.2 (with highlight / No circle was drawn. wrong)
<img src="https://user-images.githubusercontent.com/36657040/52917370-f47eb400-332d-11e9-941a-7782061f8f81.jpg" width="150">

#### v3.2.2 (without highlight / No circle was drawn. wrong)
<img src="https://user-images.githubusercontent.com/36657040/52917371-f5174a80-332d-11e9-8409-3347831a93df.jpg" width="150">

*When I switch it back to the version 3.2.1, it draws a circle  (expected behavior).

## Charts Environment

**Charts version/Branch/Commit Number:** 3.2.2 (latest version)
**Xcode version:** 10.1 (latest version)
**Swift version:** 4.2.1 (latest version)
**Platform(s) running Charts:** iOS
**macOS version running Xcode:** Mojave 10.14.3 (latest version)

## Sample Code

```Swift
let chartDataSet = LineChartDataSet(values: dataEntries, label: nil)
chartDataSet.setCircleColor(.blue)
chartDataSet.drawCirclesEnabled = true
chartDataSet.circleRadius = 1.0

lineChartView.data = LineChartData(dataSet: chartDataSet)
```

## FYI
The issues/fixes below may be related to this problem.
https://github.com/danielgindi/Charts/issues/3798
https://github.com/danielgindi/Charts/pull/3164

**Comment by lorenzOliveto** [comment]
I'm experiencing the same issue, also, adding more points the first circle is not displayed.

**Comment by staticVoidMan** [comment]
Actually, the first point doesn't have a circle. It doesn't matter if there's only one data point or more.

**Comment by jjatie** [comment]
Resolved by #3891
